frontend_vue/src/components/ui/CreditCardToken.vue

```diff
@@ -0,0 +1,55 @@
+<template>
```

Comment:
I think this is used only for the CreditCard Token
We might move this into its folder